v8-versions.json

```diff
@@ -0,0 +1,5 @@
+{
+    ">=2.5.0 <3.2.0": "5.5.372",
```

Comment:
I think it's safe to make the condition ">=2.5.0 < 3.3.0": "5.5.372" at least until we push the new v8 update. I'll take responsibility of updating the map when we are ready.